tests/integration/assessment/test_workflows.py

```diff
@@ -17,7 +26,35 @@ def test_running_real_assessment_job(ws, installation_ctx, make_cluster_policy,
     installation_ctx.__dict__['include_object_permissions'] = [f"cluster-policies:{cluster_policy.policy_id}"]
     installation_ctx.workspace_installation.run()
 
-    installation_ctx.deployed_workflows.run_workflow("assessment")
+    run_id = installation_ctx.deployed_workflows.run_workflow("assessment")
+    _assert_run_success(ws, run_id)
 
     after = installation_ctx.generic_permissions_support.load_as_dict("cluster-policies", cluster_policy.policy_id)
     assert after[ws_group_a.display_name] == PermissionLevel.CAN_USE
+
+
+@retried(on=[NotFound, InvalidParameterValue], timeout=timedelta(minutes=12))
+def test_running_real_migration_progress_job(
+    ws,
+    installation_ctx,
+    make_cluster_policy,
+    make_cluster_policy_permissions,
+) -> None:
+    """Ensure that the migration-progress workflow can complete successfully."""
+    ws_group_a, _ = installation_ctx.make_ucx_group()
```

Comment:
Why is this setup necessary for the intended test?

Reply:
I don't think it is. It's in the setup for the equivalent test that checks the `assessment` workflow, so I was verifying that this works before removing it.

To be continued…

Reply:
… and elided.